### PR TITLE
Make python 3 compatible

### PR DESCRIPTION
**Pull request by staticfox**

### CODE REVIEW

File: pysrvx/helpserv.py [hunk 1]
```diff
@@ -65,13 +65,13 @@ def stats(self, account):
         data['weekstart'] = response['data'][0][c1:c2]
 
         # Time
-        for i, key in {3: 'current', 4: 'last-1', 5: 'last-2', 6: 'last-3', 7: 'total'}.items():
+        for i, key in list({3: 'current', 4: 'last-1', 5: 'last-2', 6: 'last-3', 7: 'total'}.items()):
```
Comment (ThiefMaster) [inline pysrvx/helpserv.py]: why convert to a list?!

File: pysrvx/srvx.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def get_response(self):
                 raise NotConnected
 
             # Append the response buffer
-            self.response += tmp
+            self.response += tmp.decode('UTF-8')
```
Comment (ThiefMaster) [inline pysrvx/srvx.py]: irc is not always utf8. need to fall back to latin1 or iso-8859-1 if utf8 decoding fails, and ascii ignore (loses anything that is not ascii) if that fails too

File: pysrvx/helpserv.py (hunk 1 above)
Comment (staticfox) [reply pysrvx/helpserv.py]: `2to3` :dancer:

File: pysrvx/srvx.py [hunk 3]
```diff
@@ -125,8 +125,20 @@ def generate_token(self):
         """Return a token generated from a random number"""
         return 'GS%05d' % randint(0,65535)
 
-    def get_response(self):
+    @staticmethod
+    def decode_string(tmp):
+        decoded_string = ''
```
Comment (ThiefMaster) [inline pysrvx/srvx.py]: useless, you always assign something later - also, you can just `return` in each of the next three blocks

**Pull request merged**